src/jit/codegencommon.cpp

```diff
@@ -6919,8 +6919,13 @@ void CodeGen::genProfilingEnterCallback(regNumber initReg, bool* pInitRegZeroed)
     unsigned saveStackLvl2 = genStackLevel;
 
 #if defined(_TARGET_X86_)
-    // Important note: when you change enter probe layout, you must also update SKIP_ENTER_PROF_CALLBACK()
-    // for x86 stack unwinding
+// Important note: when you change enter probe layout, you must also update SKIP_ENTER_PROF_CALLBACK()
+// for x86 stack unwinding
+
+#if defined(UNIX_X86_ABI)
+    // CodeGen::genAlignStackBeforeCall
```

Comment:
This (and similar) comments are confusing to me. I ask: why are you referencing this function? I think you want to say something like:

```
// Manually align the stack to be 16-byte aligned. This is similar to CodeGen::genAlignStackBeforeCall(), but
// we can't use that for this helper call because <why>???
```